app/api/events.py

```diff
@@ -93,9 +93,28 @@ def get_upcoming_events(request: Request, token: AccessTokenPayload = Depends(op
     return [Event.model_validate(event) for event in upcoming_events]
 
 
+@router.get('/past-events/count')
+def get_past_events_count(request: Request, token: AccessTokenPayload = Depends(optional_authentication)):
+    """Get count of past events"""
+    db = get_database(request)
+
+    # Get current UTC datetime
+    current_datetime = datetime.now(timezone.utc)
```

Comment:
This endpoint uses `timezone.utc`, but none of the others do. This might lead to wrong results here